pkg/getty/Dockerfile

```diff
@@ -9,6 +9,9 @@ RUN apk add --no-cache --initdb -p /out \
     util-linux \
     && true
 RUN rm -rf /out/etc/apk /out/lib/apk /out/var/cache
+RUN cp /out/usr/bin/setsid /out/usr/bin/setsidu
```

Comment:
> can you just add a comment about why this is

Added lines of comment. :-)

> We should be able to remove it once init no longer uses busybox which is not that far off

Maybe. People can compose `init:` out of any containers they want. If any other uses busybox - not unreasonable - and it is later in the sequence than `getty`, it will kill it. It might be a good idea to leave it no matter what. We are guaranteed reasonably fairly this way that it will stay around. 

I think there is a bigger issue that `init:` images can trounce each other, but container images are built with the assumption that it is entirely self-contained. Probably should be a better way to handle how inits merge.